src/chunk_list.h

```diff
@@ -855,21 +852,10 @@ static inline bool chunk_is_forin(chunk_t *pc)
 }
 
 
-void set_chunk_type_real(chunk_t *pc, c_token_t tt);
-
-
-void set_chunk_parent_real(chunk_t *pc, c_token_t tt);
+void set_chunk_type(chunk_t *pc, c_token_t tt);
 
 
-#define set_chunk_type(pc, tt)      do { \
```

Comment:
I think this is a highly detrimental change. Before, these psuedo-functions would log where the call *occurs*. By making them just regular functions, we lose that. I think it would be better to retain these wrappers.

Reply:
Right, the new version gives what you wish.
Thanks